README.md

```diff
@@ -1,14 +1,14 @@
 <div align="center">
 
-  ![logo](https://raw.githubusercontent.com/KinsonDigital/Velaptor/preview/Images/velaptor-logo.png)
-</div>
+![logo](https://raw.githubusercontent.com/KinsonDigital/Velaptor/preview/Images/velaptor-logo.png)
 
+</div>
 
 <h1 style="border:0;font-weight:bold" align="center">Velaptor</h1>
 
-
```

Comment:
Please add a single empty line to this spot.

**Reason:**
I have some special processing going on with the readme file in my CICD system.
I don't think it will break anything to delete this line, but I would rather not take the chance.

Reply:
After line number 8?